attributes/_configure.rb

```diff
@@ -53,3 +53,5 @@
 
 # Default systemd file location based on default systemd unit file name
 default['splunk']['systemd_file_location'] = '/etc/systemd/system/splunk.service'
+
+default['splunk']['sslConfig']['sslPassword'] = 'password'
```

Comment:
I think this should go somewhere under the splunk/config hash for the sake of consistency.  maybe splunk/config/server/sslconfig/sslpassword or just splunk/config/sslpassword? Or somewhere in the middle...I'm open to suggestion.